components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx

```diff
@@ -77,7 +77,7 @@ export default class ManageTeamsDropdown extends React.Component {
                     <Menu.ItemAction
                         show={isTeamAdmin}
                         onClick={this.makeMember}
-                        text={Utils.localizeMessage('admin.user_item.makeMember', 'Make Member')}
+                        text={Utils.localizeMessage('admin.user_item.makeMember', 'Make Team Member')}
```

Comment:
Did you run `make i18n-extract`? I think this change will necessitate a change to ../i18n/en.json.

Reply:
This is to fix the discrepancy between the string in i18n file and the default one used in tests.